### ISSUE TITLE
Fix up CI: CI for backend

### ISSUE THREAD

**Issue by yarikoptic** [dandi/dandi-archive#1685]
In https://github.com/dandi/dandi-archive/pull/1679 I spotted that "CI for backend" is all red and apparently it was so for weeks without anyone addressing that:

![image](https://github.com/dandi/dandi-archive/assets/39889/b9634b8f-b3d3-4fca-a0de-abf6e5cbdec9)

**Comment by CodyCBakerPhD** [comment]
If it's helpful at all - I've been using an email notification as a part of the NWB Inspector health monitor (daily checks), it's proven very useful and can be modified to notify any number of people when the CI fails

https://github.com/NeurodataWithoutBorders/nwbinspector/blob/dev/.github/workflows/dailies.yml#L16-L30

**Comment by yarikoptic** [comment]
I am also getting similar notifications for a number of projects.
FWIW -- historical logs are collected by https://github.com/con/tinuous and archived on drogon at `/mnt/backup/dandi/tinuous-logs/dandi-archive/` for this repo happen you want to compare to older logs etc

**Comment by yarikoptic** [comment]
@mvandenburgh any progress on this? any new PR is hindered with redness.

**Comment by mvandenburgh** [comment]
It's a breaking change in `django-storages` (relevant issue: https://github.com/jschneier/django-storages/issues/1308). I opened [a PR](https://github.com/dandi/dandi-archive/pull/1691) to pin this dependency to the last working version, which will unblock CI while I work on getting the latest version to work.

**Comment by waxlamp** [comment]
This has been fixed.
